app/javascript/components/supplemental_application/SupplementalApplicationPage.js

```diff
@@ -332,14 +358,15 @@ const setApplicationsDefaults = (application) => {
   return applicationWithDefaults
 }
 
-const mapProperties = ({ application, statusHistory, fileBaseUrl, units, availableUnits }) => {
+const mapProperties = ({ application, statusHistory, fileBaseUrl, units, availableUnits, rentalAssistances }) => {
   return {
     application: setApplicationsDefaults(mapApplication(application)),
     statusHistory: mapList(mapFieldUpdateComment, statusHistory),
     onSubmit: (values) => updateApplicationAction(values),
     fileBaseUrl: fileBaseUrl,
     units: mapList(mapUnit, units),
-    availableUnits: mapList(mapUnit, availableUnits)
+    availableUnits: mapList(mapUnit, availableUnits),
+    rentalAssistances: rentalAssistances
```

Comment:
why doesn't rentalAssistances need to use mapList?

Reply:
because the rental assistances already come in in domain format! 🎉 thanks to our refactor work, the SOQL rental assistance service uses SOQL to fetch the rental assistances, and then translates them to domain before passing them up to the controller. that way the controller can just hand the frontend domain-formatted rental assistances. the ultimate goal here is to do away with all use of `mapList` in the frontend. and then, even further then that, once that's done and the frontend is entirely in domain, we can consider whether we'd like to convert domain format from snake case to camel case - but that's a little further off yet.